src/lm.jl

```diff
@@ -1,20 +1,35 @@
+# TODO: expand weights to include fweights, pweights, aweights
+# Example LmResp{Array{Float64,1}}(randn(5), randn(5), randn(5), randn(5))
 type LmResp{V<:FPVector} <: ModResp  # response in a linear model
     mu::V                                  # mean response
     offset::V                              # offset added to linear predictor (may have length 0)
     wts::V                                 # prior weights (may have length 0)
+    fweights::FrequencyWeights
+    pweights::ProbabilityWeights
+    aweights::AnalyticWeights
```

Comment:
I'm not sure we should carry around three different types of weights in the response object. Perhaps just a single `AbstractWeights`?